DRdigi/include/DigiSiPM.h

```diff
@@ -41,20 +39,14 @@ class DigiSiPM : public GaudiAlgorithm {
   Gaudi::Property<double> m_recovery{this, "recovery", 10., "SiPM cell recovery time in ns"}; // https://arxiv.org/abs/2001.10322
   Gaudi::Property<double> m_cellPitch{this, "cellpitch", 10., "SiPM cell size in um"};
   Gaudi::Property<double> m_afterpulse{this, "afterpulse", 0.03, "afterpulse probability"};
-  Gaudi::Property<double> m_falltimeFast{this, "falltimeFast", 6.5, "signal fast component decay time in ns"};
+  Gaudi::Property<double> m_falltimeFast{this, "falltimeFast", 6.5, "signal fast component decay time in ns"};  // This looks too fast
```

Comment:
Thanks for the comment! Indeed I was using 50 ns when Hamamatsu `S13615-1025` was the default, but changed to 6.5 ns according to the datasheet by changing the default to `S14160-1310PS`. Anyway these are configurable parameters, I've made a [configuration](https://github.com/HEP-FCC/dual-readout/blob/158c6f6714294f50d6d02c441b9f9b70247325d6/DRdigi/test/runDigi_S13615-1025.py) file for `S13615-1025` as well (the reason for changing the default is simply we purchased `S14160-1310PS` in our lab).
![cap01](https://user-images.githubusercontent.com/37464245/125894940-6e991e09-9d43-4238-a4af-4fb1d7072af1.png)